### PR TITLE
user testing guide

### PR DESCRIPTION
**Pull request by Simon-Laux**
CC @lefherz

### CODE REVIEW

File: user-testing/Desktop.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Desktop Checklist
+
+### Preparation
+
+- create or get an existing test account ready
+
+
+### Test
```
Comment (hpk42) [inline user-testing/Desktop.md]: also very important is testing the login-screen, advanced settings etc. -- that rarely gets scrutiny during development whereas cahtting/groups do get some test (dogfooding) use before releases.

**Comment by lefherz** [comment]
I see you started already - didn't get to it last week. Is contributing appreciated?

**Comment by Simon-Laux** [comment]
> Is contributing appreciated?

yeah, I just thought I'd do do the beginning so we have start.

File: user-testing/Desktop.md (hunk 1 above)
Comment (Simon-Laux) [reply user-testing/Desktop.md]: how do testers get an account that requires advanced settings?

**Comment by lefherz** [comment]
Is this only about test cases which are not checked by automated tests? 

(e.g. not sure whether disabling "prefer-encrypt: mutual" isn't already tested in the core or so... doesn't seem to be in https://github.com/deltachat/deltachat-core-rust/tree/master/tests for example)

**Comment by r10s** [comment]
> Is this only about test cases which are not checked by automated tests?

well, it is about testing using a ui - eg. group creation may fail in the _real_ ui when the create button was moved to a negative position accidentally - although group creation are tested in the core.

but it is true, for things that do not involve ui at all and are already tested in the core, there is probably no need to test them, eg. the  prefer-mutual thingie (otoh, there may still be a bug showing the correct icon - iirc pep run into that issue when showing "ok" unconditionally ;)

File: user-testing/Android.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+**Alice** will be the tester. You need 
+- an E-Mail address 
+- and the release candidate build
+- on a phone.
+- a Thunderbird installation with Enigmail or TB-AC.
+
+**Bob** is the "sparring partner" of Alice. Can be a daily-use Delta Chat
+Desktop account. You need
+- an E-Mail address
+- Delta Chat Desktop version >= 0.840.0
```
Comment (lefherz) [inline user-testing/Android.md]: So far I now stated that you need 2 accounts for testing. They send each other some messages and invite each other into groups, which is fine.

I thought it's more useful if Bob can just be a daily-use account with a stable Delta Desktop version. The main advantage is that testers only need a PC and a phone.

**Comment by lefherz** [comment]
> eg. the prefer-mutual thingie (otoh, there may still be a bug showing the correct icon - iirc pep run into that issue when showing "ok" unconditionally ;)

@r10s we show an icon for prefer-mutual? Where? afaik we still don't have an icon whether the next message will be encrypted or not.

**Pull request merged**